spec/system/admin_menage_company_values_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe 'admin menage company values', type: :system do
+  let(:admin) { create(:admin) }
+  let!(:company_value) { create(:company_value) }
+
+  before do
+    sign_in admin
+  end
+
+  it 'listing company values' do
+    visit admins_authenticated_root_path
+    click_link 'Company values'
+    expect(page).to have_content 'Title'
+    expect(page).to have_content company_value.title
+    expect(page).to have_link 'Edit'
+    expect(page).to have_link 'Destroy'
+    expect(page).to have_link 'New company value'
+  end
+
+  it 'creating company value' do
+    visit admins_authenticated_root_path
+    click_link 'Company values'
+    click_link 'New company value'
+    fill_in 'Title', with: 'Pride'
```

Comment:
check out FactoryBot `:build` or `:attributes_for` - you can use an object then instead of hardcoding new values